src/components/Form/EyeColor/EyeColor.jsx

```diff
@@ -212,17 +212,17 @@ export default class EyeColor extends ValidationElement {
                  onBlur={this.props.onBlur}
                  onFocus={this.props.onFocus}
                  >
-            <div className="eye-icon">
-				      <svg viewBox="0 0 30.86 36.84">
-					      <path id="eye-color-unknown" d="M15.43,34.25C6.91,34.25,0,27.34,0,18.82C0,10.3,6.91,3.39,15.43,3.39s15.43,6.91,15.43,15.43
-						                                    C30.87,27.34,23.95,34.25,15.43,34.25z M15.84,8.53c-3.28,0-5.73,1.41-7.46,4.28c-0.18,0.28-0.1,0.64,0.16,0.84l2.65,2.01
-						                                    c0.1,0.08,0.24,0.12,0.38,0.12c0.18,0,0.38-0.08,0.5-0.24c0.94-1.21,1.35-1.57,1.73-1.85c0.34-0.24,1-0.48,1.73-0.48
-						                                    c1.29,0,2.47,0.82,2.47,1.71c0,1.04-0.54,1.57-1.77,2.13c-1.43,0.64-3.38,2.31-3.38,4.26v0.72c0,0.36,0.28,0.64,0.64,0.64h3.86
-						                                    c0.36,0,0.64-0.28,0.64-0.64c0-0.46,0.58-1.45,1.53-1.99c1.53-0.87,3.62-2.03,3.62-5.08C23.15,11.28,19.29,8.53,15.84,8.53z
-						                                    M18.01,24.61c0-0.36-0.28-0.64-0.64-0.64H13.5c-0.36,0-0.64,0.28-0.64,0.64v3.86c0,0.36,0.28,0.64,0.64,0.64h3.86
-						                                    c0.36,0,0.64-0.28,0.64-0.64V24.61z"/>
-				      </svg>
-			      </div>
+            <div className="eye-icon unknown">
+              <svg viewBox="0 0 30.86 36.84">
+                <path id="eye-color-unknown" d="M15.43,34.25C6.91,34.25,0,27.34,0,18.82C0,10.3,6.91,3.39,15.43,3.39s15.43,6.91,15.43,15.43
```

Comment:
It's definitely a concern for maintainability. Also, huge eye sore making it harder to read through the bits you need. We were thinking of doing something like:

```jsx
<Svg src="img/eye.svg" className="eye-icon blue" dust="fairy" />
```

This way we can
 - always reference the file by path name
 - pass classes
 - and use some property (in this case `dust`) to signify whether it is necessary to use it inline for the CSS wizards